usagov/vote-gov-drupal
# Feature/VOTE-2534 Enable Japanese translation

## Jira ticket

[VOTE-2142](https://cm-jira.usa.gov/browse/VOTE-2142)

## Description

Add PO and Config files to enable and provide string translations for Japanese.

## Deployment and testing

### Pre-deploy

n/a

### Post-deploy

1. Locally, run `lando retune -y` to import config files and string translations from PO file.

### QA/Test

1.  Visit http://vote-gov.lndo.site/admin/config/regional/language and confirm that Japanese language is listed.
2. Visit http://vote-gov.lndo.site/admin/config/regional/translate
3. Do a search using the following: Translation: Japanese, Search In: Only translated strings
4. Confirm that the search returns a list of translated strings for Japanese.

## Checklist for the Developer




## Checklist for the Peer Reviewers
